extras/usd/examples/usdObj/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
+set(PXR_PREFIX pxr/usd)
 set(PXR_PACKAGE usdObj)
+unset(PXR_INSTALL_SUBDIR)
```

Comment:
I think we just need to remove this line from your change, since we don't want the usdObj plugin to build in the standard plugin location.